### PR TITLE
Loosen the AWS provider requirement

### PR DESCRIPTION
**Pull request by okgolove**
## what
* Update aws provider version requirements

## why
* 3.0 AWS provider is already here and we should be able to use the module with it

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```
Comment (jamengual) [inline versions.tf:2]: please just use 
```
required_version = ">= 0.12"
```

**Comment by joe-niland** [comment]
@aknysh @jamengual do you think we should use lower bound pinning on the null, template and local providers also ?

**Comment by aknysh** [comment]
> @aknysh @jamengual do you think we should use lower bound pinning on the null, template and local providers also ?

we do it everywhere, and out tests check for all provider pinning, so I'd say let's continue doing it

File: versions.tf [hunk 2]
```diff
@@ -1,8 +1,8 @@
 terraform {
-  required_version = ">= 0.12.0, < 0.14.0"
+  required_version = ">= 0.12.0"
 
   required_providers {
-    aws      = "~> 2.0"
+    aws      = ">= 2.0"
     template = "~> 2.0"
```
Comment (aknysh) [inline versions.tf:6]:
```suggestion
    template = ">= 2.0"
```

File: versions.tf [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
     null     = "~> 2.0"
```
Comment (aknysh) [inline versions.tf:7]:
```suggestion
    null     = ">= 2.0"
```

File: versions.tf [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
     local    = "~> 1.3"
```
Comment (aknysh) [inline versions.tf:8]:
```suggestion
    local    = ">= 1.3"
```

**Comment by okgolove** [comment]
@aknysh fixed!

**Comment by aknysh** [comment]
@okgolove thanks
sorry, forgot to mention it, but please do the same changes in all of the examples
https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/examples/complete/versions.tf
https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/examples/with_cognito_authentication/versions.tf
https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/examples/with_google_oidc_authentication/versions.tf
https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/examples/without_authentication/versions.tf

**Comment by okgolove** [comment]
@aknysh done.

**Review by aknysh — APPROVED** [review]

**Pull request merged**